standard/ieee/draft/802.1/Qcw/ieee802-dot1q-preemption.yang

```diff
@@ -29,21 +29,9 @@ module ieee802-dot1q-preemption {
   description
     "This module provides for management of IEEE Std 802.1Q Bridges
     that support Frame Preemption.";
-  revision 2019-06-24 {
```

Comment:
It's strongly recommended, and pretty much mandatory, to have a revision history in models. Any reason why it is being completely removed?

Reply:
I tend to agree. However since revision history (during draft phase) will be lost after the publication I stopped updating it. And not wanting to have an outdated revision, I removed it.